src/vcUnitConversion.cpp

```diff
@@ -61,3 +62,137 @@ double vcUnitConversion_ConvertTemperature(double sourceValue, vcTemperatureUnit
 
   return celciusVal;
 }
+
+vcTimeReferenceData vcUnitConversion_ConvertTimeReference(vcTimeReferenceData sourceValue, vcTimeReference sourceReference, vcTimeReference requiredReference)
+{
+  vcTimeReferenceData result = {false, 0.0};
```

Comment:
You should be able to leave this as `{}` to initialise all members to 0 - if the union is rearranged code like this won't break. The first member is also a sub-object and you technically have to initialise it `{false, { 0.0 }}`, which is why this doesn't build on Clang.